### DISCUSSION TITLE
Scrollable Table Pane With Fixed Headers

### DISCUSSION THREAD

**Discussion by jrees20** [dxc-technology/halstack-react#492] [Ideas] [answered]
For tables that are displaying several rows, it is desirable that the user can scroll the table pane leaving the headings at the top of the data so that the user can always see the headings.  This should carry over to each next page on the pagination bar. 

![image](https://user-images.githubusercontent.com/93621822/148109652-0cf590ce-bb5a-465d-b5c7-3206fe465d92.png)

As the item per page increases, we would like to see a scroll bar for just the data  leaving the purple headings always in view.

**Comment by aweell** [comment, marked as answer]
Hi @jrees20!

Thanks for reaching, we have plans to review the design of our table component in the current release:

https://github.com/dxc-technology/halstack/issues/31
